ui/app/AppLayouts/Profile/ProfileLayout.qml

```diff
@@ -24,10 +25,14 @@ StatusAppTwoPanelLayout {
     }
 
     QtObject {
-        id: _internal
-        readonly property int contentMaxWidth: 624
-        readonly property int contentMinWidth: 450
-        property int profileContentWidth: Math.max(contentMinWidth, Math.min(profileContainer.width * 0.8, contentMaxWidth))
+        id: d
```

Comment:
nit: what is `d` ? maybe something more meaningful?

Reply:
@alaibe this is a common Qt pattern to hide implementation details. See here for more details: https://wiki.qt.io/D-Pointer#The_d-pointer

Reply:
@alaibe in all qml components defined by Qt `d` is used as `id` of `QtObject` containing private properties (I think it stands for "details"), check [here](https://doc.qt.io/archives/qt-4.8/qml-coding-conventions.html#private-properties).